test/jdk/java/lang/invoke/CallSiteTest.java

```diff
@@ -92,7 +93,7 @@ static void test(boolean shouldThrow) {
         }
     }
 
-    static class MyCCS extends ConstantCallSite {
+    public static class MyCCS extends ConstantCallSite {
```

Comment:
Otherwise it thrown an java.lang.IllegalAccessError:

> Caused by: java.lang.IllegalAccessError: failed to access class test.java.lang.invoke.CallSiteTest$MyCCS from class test.java.lang.invoke.CallSiteTest (test.java.lang.invoke.CallSiteTest$MyCCS is in unnamed module of loader 'app'; test.java.lang.invoke.CallSiteTest is in unnamed module of loader indify.Indify$Loader @234ba0f8)